### PR TITLE
Adding @KeithMon's Rak connector diagram to 19003/7

### LINKED ISSUE #1029: Document RAK connector Types
This is commonly asked, perhaps we can use @KeithMon 's gorgeous image in the docs
image

### PR DESCRIPTION
**Pull request by exotime**
Also transcribed key voltage and connector specifications into text for accessibility.

Fixes #1029.

### CODE REVIEW

**Review by pdxlocations — APPROVED** [review]

**Comment by pdxlocations** [comment]
Thank you!  Any chance you'd like to add the [5005-O and 19001?](https://github.com/meshtastic/meshtastic/issues/1029#issuecomment-2408445987)

File: docs/hardware/devices/rak-wireless/wisblock/base-boards.mdx [hunk 1]
```diff
@@ -71,15 +71,26 @@ Further information on the RAK5005-O can be found on the [RAK Documentation Cent
     - (x1) Reset Button
     - To add a user button, you can utilize the AIN1 pin, which is exposed as pin 31 in the firmware.
   - **Connectors**
-    - Connector for 3.7v LiPo battery (with charge controller)
-    - Connector for 5v solar panel (max 5.5v)
+    - JST PHR-2 connector for 3.7v LiPo battery.
```
[the remark this answers is not in the document]
Comment (exotime) [reply docs/hardware/devices/rak-wireless/wisblock/base-boards.mdx:89]: Addressed in [c7dc3fe](https://github.com/meshtastic/meshtastic/pull/1499/commits/c7dc3fe52b1b9b295ddd8164e00a68adffaa1b7a).

File: docs/hardware/devices/rak-wireless/wisblock/base-boards.mdx [hunk 2]
```diff
@@ -103,15 +114,26 @@ Further information on the RAK19007 can be found on the [RAK Documentation Cente
     - (x1) Reset Button
     - There is a 3rd party sensor module that breaks out the user button, available [here](https://www.etsy.com/de/listing/1254963521/mini-io-expansion-board-for-rak-wireless).
   - **Connectors**
-    - Connector for 3.7v LiPo battery (with charge controller)
-    - Connector for 5v solar panel (max 5.5v)
+    - JST PHR-2 connector for 3.7v LiPo battery.
```
[the remark this answers is not in the document]
Comment (exotime) [reply docs/hardware/devices/rak-wireless/wisblock/base-boards.mdx:136]: Addressed in [c7dc3fe](https://github.com/meshtastic/meshtastic/pull/1499/commits/c7dc3fe52b1b9b295ddd8164e00a68adffaa1b7a).

File: docs/hardware/devices/rak-wireless/wisblock/base-boards.mdx [hunk 3]
```diff
@@ -29,25 +29,40 @@ The RAK5005-O is no longer in production. It is recommended to use the RAK19007
...
-  - (x1) Reset Button
-  - It may be possible to add a user button using the [13002 IO module](https://store.rakwireless.com/collections/wisblock-interface/products/adapter-module-rak13002).
-- **Connectors**
-- Connector for 3.7v LiPo battery (with charge controller)
-- Connector for 5v solar panel (max 5.5v)
-- I<sup>2</sup>C, UART, GPIOs and analog input accessible with solder contacts
-- Micro USB port for debugging and power
-- **Screen Support**
-- OLED screen support (OLED screen sold separately)
+  - **Slots**
```
[the remark this answers is not in the document]
Comment (exotime) [reply docs/hardware/devices/rak-wireless/wisblock/base-boards.mdx:32]: No, this change made it consistent with the other hardware tabs which had it originally. 

Please try and provide all your feedback at once. This PR has already been _far_ too much effort, and you are not encouraging me to continue contributing.

**Pull request merged**